src/io/backend_cpal.rs

```diff
@@ -89,7 +90,23 @@ impl AudioBackend for CpalBackend {
     where
         Self: Sized,
     {
-        let mut builder = StreamConfigsBuilder::new();
+        let host = cpal::default_host();
+        log::info!("Host: {:?}", host.id());
```

Comment:
Maybe this should be in the "None" branch, as this is not used in the other branch, the log might be confusing if sink_id is set

Reply:
Good point.
It is a bit sloppy, the other branch eventually also calls into the default host as well, so the statement is not wrong per se.